www/templates/control/main-control.html

```diff
@@ -8,47 +8,71 @@
-      <div class="item row">
-        <div class="col"> {{settings.auth.email}} </div>
-        <button class="col col-25 button button-positive" ng-click="returnToIntro()"> Login </button>
-      </div>
+  </div> -->
+
+  <ion-content style="background-color: #262A30;">
+<!--     <div style="text-align: center;">
+      <img src="img/avatar_1.png" ng-style="getAvatarStyle()">
+    </div> -->
+    
+    <div style="background-color: white; height: 50px;">
+      <div style="padding: 15px 15px; float: left;">{{settings.auth.email}}</div>
+      <div ng-click="returnToIntro()" ng-style="getIconButtonStyle('E0A1A1')"><i class="icon ion-log-out" ng-style="getIconStyle()"></i></div>
     </div>
-    <div class="card">
-      <div class="item item-divider">
-        State
-      </div>
-      <div class="item row">
-        <div class="col"> {{settings.collect.state}} </div>
-        <button class="col col-25 button button-positive" ng-click="forceState()"> Force state </button>
-      </div>
+    
+    <div style="background-color: white; height: 50px; margin-top: 0.5px;">
+      <div style="padding: 15px 15px; float: left;">Developer zone</div>
+      <div ng-click="toggleCollection()" ng-style="getIconButtonStyle('D6C780')"><i ng-class="getExpandButtonClass()" ng-style="getIconStyle()" id="expandButton"></i></div>
     </div>
-    <div class="card">
-      <div class="item item-divider">
-        Connection
+    
+    
+    <div class="item-accordion" ng-show="collectionExpanded()">
+      <div style="background-color: white; height: 50px; margin-top: 0.5px;">
+        <div style="padding: 15px 15px; float: left;">{{parseState(settings.collect.state)}}</div>
+        <div ng-click="forceState()" ng-style="getIconButtonStyle('CCCCCC')"><i class="ion-edit" ng-style="getIconStyle()"></i></div>
       </div>
-      <div class="item">
-        {{settings.connect.url}}
+      <div style="background-color: white; height: 50px; margin-top: 0.5px;">
+        <div style="padding: 15px 15px; float: left;">Check log</div>
+        <div ng-click="showLog()" ng-style="getIconButtonStyle('CCCCCC')"><i class="ion-ios-arrow-right" ng-style="getIconStyle()"></i></div>
+      </div>      
+      <div style="background-color: white; height: 50px; margin-top: 0.5px;">
+        <div style="padding: 15px 15px; float: left;">Check sensed data</div>
+        <div ng-click="showSensed()" ng-style="getIconButtonStyle('CCCCCC')"><i class="ion-ios-arrow-right" ng-style="getIconStyle()"></i></div>
+      </div>   
+      <div style="background-color: white; height: 50px; margin-top: 0.5px;">
+        <div style="padding: 15px 15px; float: left;">Check map</div>
+        <div ng-click="showMap()" ng-style="getIconButtonStyle('CCCCCC')"><i class="ion-ios-arrow-right" ng-style="getIconStyle()"></i></div>
+      </div>   
+      <div style="background-color: white; height: 50px; margin-top: 0.5px;">
+        <div style="padding: 15px 15px; float: left;">Collection</div>
```

Comment:
Why are we using more `px`? Shouldn't we be using `em` or relative widths instead?